### PR TITLE
CLDR-14055 Add annotations for new locales

### PR DESCRIPTION
**Pull request by andrew-z**
CLDR-14055 

##### Checklist

- [x] Issue filed: https://unicode-org.atlassian.net/browse/CLDR-_____
- [x] Updated PR title and link in previous line to include Issue number

### CODE REVIEW

**Comment by andrew-z** [comment]
Several unit tests are failing - need to learn how to mitigate

**Comment by andrew-z** [comment]
@macchiati @Kristi27 
I've prepared a first draft of new locale annotations. I solved my issues with Java so I could run tests locally, but doing so lead to a problem: for 3 locales (nso, quc, tn) tests fail because these locales are not present in common\main folder. They are. however, present in seed\main, but I don't understand what I should do to make the test happy with that.

Could you please advise?

**Comment by andrew-z** [comment]
> the files look good. However, some of them are in the wrong place. If the main locale is in seed/main/, such as nso.xml, then the annotations have to be in seed/annotations

Thanks Mark.
I didn't realize there is annotations folder under seed as well... 
Would putting those seed files from me there be enough? I don't understand what's the difference between "seed" and the rest.

**Comment by macchiati** [comment]
Practically speaking, you don't need to know the difference; just that if
you see an XXX.xml in seed/main, then the annotation file XXX.xml must go
into seed/annotations, not in main/annotations.

Mark

**Comment by Kristi27** [comment]
@macchiati Do the  changes look good? 
Are we going to merge this for v38?

File: common/annotations/mt.xml [hunk 1]
```diff
@@ -0,0 +1,2543 @@
...
+		<annotation cp="💪" type="tts">biceps bil-muskoli</annotation>
+		<annotation cp="👂">ġisem</annotation>
+		<annotation cp="👂" type="tts">widna</annotation>
+		<annotation cp="👃">ġisem</annotation>
+		<annotation cp="👃" type="tts">imnieħer</annotation>
+		<annotation cp="🧠">intelliġenti</annotation>
+		<annotation cp="🧠" type="tts">moħħ</annotation>
+		<annotation cp="👀">wiċċ | għajn</annotation>
+		<annotation cp="👀" type="tts">għajnejn</annotation>
+		<annotation cp="👁">ġisem</annotation>
```
Comment (srl295) [inline common/annotations/mt.xml:380]: should be għajn not ġisem (body)

File: common/annotations/mt.xml [hunk 2]
```diff
@@ -0,0 +1,2543 @@
...
+		<annotation cp="👁" type="tts">għajn</annotation>
+		<annotation cp="👅">ġisem</annotation>
```
Comment (srl295) [inline common/annotations/mt.xml:382]: ilsien (tongue) not ġisem (body)

File: common/annotations/mt.xml (hunk 1 above)
Comment (srl295) [inline common/annotations/mt.xml:380]:
```suggestion
		<annotation cp="👁">għajn</annotation>

```

File: common/annotations/mt.xml (hunk 2 above)
Comment (srl295) [inline common/annotations/mt.xml:382]:
```suggestion
		<annotation cp="👅">ilsien</annotation>

```

as per the tts

File: common/annotations/mt.xml [hunk 3]
```diff
@@ -0,0 +1,2543 @@
...
+		<annotation cp="👅" type="tts">ilsien</annotation>
+		<annotation cp="👄">xufftejn</annotation>
+		<annotation cp="👄" type="tts">ħalq</annotation>
+		<annotation cp="👶">żagħżugħ | tarbija</annotation>
```
Comment (srl295) [inline common/annotations/mt.xml:386]:
```suggestion
		<annotation cp="👶">żagħżugħ | tarbija</annotation>

```

this is more of a tarbija (baby) than a żagħżugħ (youth/youngster)

**Comment by andrew-z** [comment]
@srl295 I've fixed Maltese as you suggested. 
By the way, "single-commit" check encourages me to do force push every time, so I don't understand how can I accept proposed changes (since that would create a second commit). Or maybe I'm using GitHub wrong? 🤔 (my experience is mostly with AzureDevOps...)

**Comment by andrew-z** [comment]
@srl295 As for uniqueness check - I am not confident I can fix all of that 😐 Most of the duplicates (leaving out quc for now) are because of 🆙 and 🔼 having same tts name, which is a very known issue I see (even pointed out in [this page](http://cldr.unicode.org/translation/characters-emoji-symbols/short-names-and-keywords#TOC-Unique-Names)). I am trying to use my linguistic intuition for some cases by replacing "button up" in 🆙 with just "up", but of course I have no confidence I'm doing it correctly. What would be the way to proceed with this in general? Is it still okay to check in with some questionable translations and then get them corrected?

**Comment by Kristi27** [comment]
@andrew-z for timing purposes (we are creating the maintenance branch shortly), @srl295 will do the following:
1. Merge 
2. Run DAIP
3. Generate Derived

**Comment by srl295** [comment]
CLDRModify (DAIP) done in #691

**Pull request merged**